lib/chefspec/coverage.rb

```diff
@@ -67,7 +69,26 @@ def add_filter(filter = nil, &block)
 
       true
     end
-
+    #
+    # Change the template for reporting of converage analysis.
+    #
+    # @param [string] path
+    #   The template file to use for the output of the report
+    #
+    # @return [true]
+    #
+    def set_template(file = 'human.erb')
+      [
+        ChefSpec.root.join('templates', 'coverage', file),
+        File.join(Dir.pwd,file)
+      ].each do |template|
+        if File.exist? template
+          @erb = Erubis::Eruby.new(File.read(template))
```

Comment:
As per [this comment](https://github.com/sethvargo/chefspec/pull/694/files#r58626484), I think this should just set `@template` to the right path, and then we do the ERB processing in the actual report.